### ISSUE TITLE
Hyphenated compound surnames

### ISSUE THREAD

**Issue by JaMeZ-B** [plk/biber#272]
Consider the following MWE:

```latex
\documentclass{scrartcl}
\usepackage[utf8]{inputenc}
\usepackage{filecontents}
\begin{filecontents*}{\jobname.bib}
	@article{GRW_high1,
		author = {Galatius, S{\o}ren and Randal-Williams, Oscar}, % REMOVE HYPHEN
		title = {Homological stability for moduli spaces of high dimensional manifolds. {I}},
		journal = {Journal of the American Mathematical Society},
		volume = {31},
		year = {2018}
	}
\end{filecontents*}


\usepackage[%
	backend=biber,
	style=alphabetic,
	]%
{biblatex}


% ---- TOGGLE THIS

\DeclareLabelalphaNameTemplate{
	\namepart[use=true, pre=true, strwidth=1, compound=true]{prefix}
	\namepart[compound=true]{family}
}

\addbibresource{\jobname.bib}

\begin{document}
	\cite{GRW_high1}
	\printbibliography
\end{document}
```
This generates the label `[GR18]`, which _should_ be `[GRW18]` I think, since according to the manual (p. 201):

> For static (non-varwidth) disambiguation in `\DeclareLabelalphaTemplate`, nameparts separated by whitespace or **hyphens** (compound names) as separate names for label generation. This means that when forming a label out of, for example the
> surname ‘Ballam Forsyth’ with a 1 character, left-side substring, this name wouldgive ‘BF’ with `compound=true` and ‘B’ with `compound=false`.

Also: 
* GRW is created in this setting, when the hyphen is replaced by a space
* Generating GRW is the standard behaviour of the BibTeX's `alpha.bst`

Because of the manual entry I consider this a bug – but even otherwise it would be the more sensible behaviour to treat the label creation of compound surnames, I think.

**Comment by plk** [comment]
Thank you for reporting this, you are quite right that this was a bug. It should be fixed in DEV version 2.13 on SF with regression tests in place.

**Comment by plk** [comment]
Released in version 2.13 which is now live.
